File: back-end/capstone back-end/src/main/java/it/epicode/capstone/Models/ResponsesDTO/AccessTokenRes.java
package it.epicode.capstone.Models.ResponsesDTO;

import io.swagger.v3.oas.annotations.tags.Tag;
import it.epicode.capstone.Models.Entities.User;
import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
@Tag(name = "Access Token Response")
public class AccessTokenRes {
    private String accessToken;
    private String dateOfBirth;
    private String name;
    private String role;
    private String surname;
    private String username;
}

File: front-end/src/app/api/models/access-token-res.ts
/* tslint:disable */
/* eslint-disable */
export interface AccessTokenRes {
  accessToken: string;
  user: UserToken

}

export interface UserToken{
  dateOfBirth?: string;
  name?: string;
  logoProfile?:string
  role?: string;
  surname?: string;
  username?: string;
}
